earth_enterprise/rpms/build.gradle

```diff
@@ -55,18 +66,9 @@ def openGeeVersion =
     findAll { !it.isEmpty() }. // Skip empty lines.
     findResult {
         it // The first string that remains is the version string.
-    }
-
-def stagedInstall_path = '/tmp/fusion_os_install'
-if (new File('build.properties').exists()) {
-    def props = new Properties()
-    new File('build.properties').withInputStream {
-        props.load(it)
-    }
-    if(props["staged"]) {
-        stagedInstall_path = props["staged"]
-    }
-}
+    }.
+    replaceAll(/-.*release/, '').
+    replaceAll('-', '.')
```

Comment:
Moving the discussion here, since the latest change ate the comments:
tst-ccamp an hour ago  Collaborator
Looking at this a little more, I think for the sake of proper upgrades, we may have to truncate the "-2" or move it instead of just replacing the '-' with a '.'. Because it will consider 5.2.1.2 to be a newer version than 5.2.1 no matter what else comes after it.
      @tst-jblair
tst-jblair an hour ago  Collaborator
I mentioned this to @tst-jcupp, but he decided that the larger priority was to match the RPM guidance in the Version Description document and get something working (given that there's some current process that uninstalls the existing version anyway?). So, this PR is designed to meet the Version document as closely as possible (for RPMS - also ignoring .deb for now).